css/Components/ShowMore.scss

```diff
@@ -37,6 +38,14 @@
   max-height: 400px;
   pointer-events: none;
 
+  background-image: linear-gradient(to bottom, rgba(8, 19, 64, 0), $Color-deep-cove);
+
+  @include variablesSupported {
+    background-image: linear-gradient(to bottom, rgba(8, 19, 64, 0), var(--color-background));
+
+    --color-background: #{$Color-deep-cove};
```

Comment:
please declare this at the root of the component, and rename it to something like `show-more-overlay-background`